### PR TITLE
Port docs.influxdata.com#2938 to v1-merge branch

### PR DESCRIPTION
**Pull request by pierwill**
See https://github.com/influxdata/docs.influxdata.com/pull/2938.

### CODE REVIEW

File: content/telegraf/v1.15/administration/windows_service.md [hunk 1]
```diff
@@ -8,41 +8,59 @@ menu:
...
+https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7#with-administrative-privileges-run-as-administrator).
+{{% /note %}}
 
+1. Download the Telegraf binary and unzip its contents to `C:\Program Files\InfluxData\Telegraf`.
+2. In PowerShell, run the following as an administrator:
+   ```powershell
+   > cd "C:\Program Files\InfluxData\Telegraf"
+   > .\telegraf.exe --service install --config "C:\Program Files\InfluxData\Telegraf\telegraf.conf"
    ```
+   When installing as service in Windows, always double check to specify full, correct path of the config file.
```
Comment (sanderson) [inline content/telegraf/v1.15/administration/windows_service.md:36]: When installing Telegraf as a Windows service, provide the **absolute path** of the Telegraf configuration file.

File: content/telegraf/v1.15/administration/windows_service.md [hunk 2]
```diff
@@ -8,41 +8,59 @@ menu:
...
-| `telegraf.exe --service install`   | Install telegraf as a service |
-| `telegraf.exe --service uninstall` | Remove the telegraf service   |
-| `telegraf.exe --service start`     | Start the telegraf service    |
-| `telegraf.exe --service stop`      | Stop the telegraf service     |
+1. Create a directory for configuration snippets: `C:\Program Files\Telegraf\telegraf.d`
+2. Include the `--config-directory` option when registering the service:
+   ```
+   > C:\"Program Files"\Telegraf\telegraf.exe --service install --config C:\"Program Files"\Telegraf\telegraf.conf --config-directory C:\"Program Files"\Telegraf\telegraf.d
+   ```
+-->
```
Comment (sanderson) [inline content/telegraf/v1.15/administration/windows_service.md:60]: Is there a reason this is commented out? I support coming for this later?

File: content/telegraf/v1.15/administration/windows_service.md [hunk 3]
```diff
@@ -8,41 +8,59 @@ menu:
...
+
+{{% note %}}
+## Logging and troubleshooting
+When Telegraf runs as a Windows service, Telegraf logs messages to Windows event logs.
+If the Telegraf service fails on start, view error logs by selecting **Event Viewer**→**Windows Logs**→**Application**.
```
Comment (sanderson) [inline content/telegraf/v1.15/administration/windows_service.md:65]: ...fails ~on~ to start.

File: content/telegraf/v1.15/administration/windows_service.md (hunk 2 above)
Comment (pierwill) [reply content/telegraf/v1.15/administration/windows_service.md:60]: I had not been able to adequately test this yet in Windows, so leaving it commented for now.

**Pull request merged**